egg.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "drash-middleware",
+  "description": "A middleware library for Drash.",
+  "version": "0.4.1",
+  "stable": true,
+  "repository": "https://github.com/drashland/deno-drash-middleware",
+  "files": [
+    "./dexter/**/*",
+    "./deps.ts",
+    "./README.md",
+    "./logo.png",
+    "LICENSE",
+    "./paladin/**/*"
+  ]
```

Comment:
Why not simply add `**`. If we add more middleware we don't have to add them manually here.

Reply:
Good idea lol :p

Reply:
https://github.com/drashland/deno-drash-middleware/pull/37/commits/9beab8212a7224076e4a8464095f9ccbcc3091ff